nifi-docs/src/main/asciidoc/administration-guide.adoc

```diff
@@ -2338,15 +2442,15 @@ We add the following line anywhere in this file in order to tell the NiFi JVM to
 [source]
 java.arg.15=-Djava.security.auth.login.config=./conf/zookeeper-jaas.conf
 
-Finally we need to update `nifi.properties` to ensure that NiFi knows to apply SASL specific ACLs for the Znodes it will create in Zookeeper for cluster management.
+Finally we need to update _nifi.properties_ to ensure that NiFi knows to apply SASL specific ACLs for the Znodes it will create in Zookeeper for cluster management.
 To enable this, in the `$NIFI_HOME/conf/nifi.properties` file and edit the following properties as shown below:
```

Comment:
Change path _$NIFI_HOME/conf/nifi.properties_ to italics?

Reply:
yes